testsuite/forge.py

```diff
@@ -1254,56 +1241,31 @@ def test(
     """Run a forge test"""
     shell = FakeShell() if dry_run else LocalShell(verbose == "true")
     git = Git(shell)
-    filesystem = LocalFilesystem()
+    filesystem = FakeFilesystem() if dry_run else LocalFilesystem() 
     processes = FakeProcesses() if dry_run else SystemProcesses()
     time = FakeTime() if dry_run else SystemTime()
 
-    if dry_run:
-        aws_account_num = "1234"
-    # Pre flight checks
-    else:
-        try:
-            assert_aws_auth(shell)
-            aws_account_num = get_aws_account_num(shell)
-        except Exception:
-            update_aws_auth(shell, aws_auth_script)
-            aws_account_num = get_aws_account_num(shell)
-
-    if aws_auth_script and aws_token_expiration and not dry_run:
-        assert_aws_token_expiration(
-            os.getenv("AWS_TOKEN_EXPIRATION", aws_token_expiration)
-        )
-
-    assert aws_account_num is not None, "AWS account number is required"
+    aws_account_num = None
+    try:
+        aws_account_num = get_aws_account_num(shell)
```

Comment:
This value isn't used anywhere?

Reply:
This gets passed into the ForgeContext and used by the k8s runner